chainerrl/agents/categorical_double_dqn.py

```diff
@@ -24,10 +21,16 @@ def _compute_target_values(self, exp_batch):
         batch_rewards = exp_batch['reward']
         batch_terminal = exp_batch['is_state_terminal']
 
-        with chainer.using_config('train', False), state_kept(self.q_function):
```

Comment:
I think we should keep `with chainer.using_config('train', False)`

Reply:
I agree. It only makes a difference when you use certain links or functions (e.g. BatchNormalization and dropout), but I don't have any specific reason for changing the current behavior.